packages/hub/node-tests/routes/config-test.ts

```diff
@@ -0,0 +1,29 @@
+import { setupHub } from '../helpers/server';
+
+describe('GET /api/config', function () {
+  let { request } = setupHub(this);
+
+  it('returns config information', async function () {
+    await request()
+      .get('/api/config')
+      .set('Accept', 'application/vnd.api+json')
+      .set('Content-Type', 'application/vnd.api+json')
+      .expect(200)
+      .expect({
+        data: {
+          type: 'config',
+          attributes: {
+            web3: {
+              layer1Network: 'kovan',
+              layer1RpcNodeHttpsUrl: 'https://infuratest.test/abc123/',
+              layer1RpcNodeWssUrl: 'wss://infuratest.test/ws/abc123/',
+              layer2Network: 'sokol',
+              layer2RpcNodeHttpsUrl: 'https://humorme.test/abc123/',
+              layer2RpcNodeWssUrl: 'wss://humorme.test/abc123/',
```

Comment:
something to keep in mind: in the not to distant future we'll need to support multiple layer 2 networks (avalanche, polygon, etc). probably don't need to change this now, but something to just keep in the back of our heads

Reply:
Yeah, this made my head spin too much to solve now since we have a lot of assumptions about layer1/layer2right now.